pkg/controller/acicontainersoperator.go

```diff
@@ -469,6 +535,39 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 			}
 		}
 	}
+
```

Comment:
Do we have to enable the creation of accprovisioninput CR once acicnioperator CR is created or both are independent? If in case acicnioperator CR is not created, do we need to go ahead create the accprovisioninput ?

Reply:
Thanks for pointing it out. 
I have introduced a new variable to ensure that we don't work on the second CR till acicnioperator CR is created https://github.com/noironetworks/aci-containers/blob/f928ebfc418d23bbcecd1236881d07235da3fc5c/pkg/controller/acicontainersoperator.go#L505